scipy/interpolate/_interpolate.py

```diff
@@ -176,8 +178,9 @@ class interp2d:
     of 0. If more control over smoothing is needed, `bisplrep` should be
     used directly.
 
-    The new data points to interpolate `xnew` and `ynew` have to be
-    sorted by increasing order.
+    The coordinates of the data points to interpolate `xnew` and `ynew`
+    have to be sorted by ascending order. If it is not the case
+    `RegularGridInterpolator` should be used instead.
```

Comment:
While we're at it, let's strengthen the message: `interp2d` is legacy and is not recommended for use in new code. New code should use `RegularGridInterpolator` instead.

Reply:
should be resolved by https://github.com/scipy/scipy/pull/16511/commits/a50b21476e97a0e2b809d9c777eef0cce2459c5c